shell/lib/docker.sh

```diff
@@ -172,3 +173,10 @@ run_docker() {
   docker "$@"
   set +x
 }
+
+# docker_manifest_images retrieves the list of images from a manifest file.
+docker_manifest_images() {
+  local manifest="$1"
+
+  "$YQ" -r 'keys[]' "$manifest"
```

Comment:
**nitpick:** Not much value to using a local variable for a one-liner is there?

Reply:
If bash function args were named, sure.